nxdrive/osi/extension.py

```diff
@@ -34,6 +34,16 @@ class Status(Enum):
     UNSYNCED = 6
 
 
+icon_status = {
```

Comment:
Perhaps could you add a comment like to explain why this is needed? Something like "Map status to emblem basename, used on GNU/Linux", maybe?